src/Compatibility/Microsoft.DotNet.PackageValidation/BaselinePackageValidator.cs

```diff
@@ -16,17 +16,16 @@ namespace Microsoft.DotNet.PackageValidation
     /// Validates that no target framework / rid support is dropped in the latest package.
     /// Reports all the breaking changes in the latest package.
     /// </summary>
-    public class BaselinePackageValidator
+    internal class BaselinePackageValidator
```

Comment:
If we wanted these object models to be used to build an App or a tool on top of them we would need them to be public.

What if we leave the object models to receive an `ILogger` and then you could add a class that is internal and implements `ILogger` and then `Log.LogError` redirects to `Log.LogNonSdkError`? 

Then if we use these object models for a dotnet tool or another app on top of them to validate packages, those apps can implement their own `ILogger` classes and log to the console or somewhere else.